streetcrm/admin.py

```diff
@@ -552,8 +551,24 @@ def advanced_search_do(self, request, form, export):
                                "result_count": count_set['result_count']
             }
         else:
+            search_params = []
+            for key, value in data.items():
+                if isinstance(value, datetime):
+                    search_params.append('{} = {}'.format(key, value.strftime('%Y-%m-%d')))
+                elif isinstance(value, str) and len(value):
+                    search_params.append('{} = {}'.format(key, value))
+                elif isinstance(value, bool) and value:
+                    search_params.append(key)
+                elif hasattr(value, '__iter__') and len(value):
+                    search_params.append('{} = {}'.format(key, ','.join([str(v) for v in value])))
+                elif hasattr(value, 'pk'):
+                    search_params.append('{} = {}'.format(key, str(value)))
+
+            search_header = ', '.join(search_params)
+
             results_package = {"form": form,
                                "search_results": results,
+                               "search_header": search_header,
```

Comment:
Right, exactly.  Thanks, @kfogel.  

Now that I've had a chance to look at it more closely, it might be difficult to assemble in `export_search`, since we don't pass `data` to `export_search`, so we wouldn't be able to use the `for` loop approach.  We could abstract that into a different function, called from `advanced_search_do`, to assemble the query parameters into a string.  

What do you think about naming the parameters something other than their keys in the form data?  For example, `search_model` is a little unintuitive to the everyday user.  That might make this more complicated than we want, but I'm curious about your thoughts.